app/controllers/candidate_interface/gcse/grade_controller_concern.rb

```diff
@@ -0,0 +1,38 @@
+module CandidateInterface
+  module Gcse
+    module GradeControllerConcern
```

Comment:
I think the concern is a good interim step :+1: . I suspect we may end up refactoring a lot of it away as the feature progresses.